tests/client/object_connectivity_moni.c

```diff
@@ -334,11 +332,25 @@ lwm2m_object_t * get_object_conn_m()
         memset(connObj, 0, sizeof(lwm2m_object_t));
 
         /*
-         * It assign his unique ID
-         * The 3 is the standard ID for the mandatory object "Object device".
+         * It assigns his unique ID
```

Comment:
I think this comment have a really low value:
by reading the code we know we are assigning an ID
just remove it :)